templates/tonic-web-server-ingress.yaml

```diff
@@ -11,6 +11,9 @@ metadata:
     nginx.ingress.kubernetes.io/ssl-passthrough: "true"
     nginx.ingress.kubernetes.io/service-upstream: "true"
     nginx.ingress.kubernetes.io/default-backend: "tonic-web-server"
+    nginx.ingress.kubernetes.io/proxy-read-timeout: "3600" # 1 hour
+    nginx.ingress.kubernetes.io/proxy-send-timeout: "3600" # 1 hour
+    nginx.ingress.kubernetes.io/proxy-body-size: "100m" #100 MB
```

Comment:
I don't think this is large enough. Sometimes that table payload can be > 300mb. Not sure if this is for request too but with the new file upload database connector, it is conceivable that someone could try to upload several gigabytes.